tests/test_multidict.py

```diff
@@ -1,82 +1,92 @@
+import operator
 import sys
+from functools import reduce
+
 import pytest
 
 import multidict
 
-from multidict._compat import USE_CYTHON
 
-if USE_CYTHON:
-    from multidict._multidict import (
-        MultiDictProxy,
-        MultiDict,
-        CIMultiDictProxy,
-        CIMultiDict,
-        istr
-    )
+def chained_ctor(_multidict_module, impls):
```

Comment:
I cannot guess what this function name stands for. Could you plz think of its improvement? Explicit is better, than implicit.

Oh.. and it would be great it you could add docstrings.

Reply:
Sure, will do. Any preference on name?